### DISCUSSION TITLE
API to get all Children of a Parent Inscription

### DISCUSSION THREAD

**Discussion by shanerich** [ordinals/ord#3773] [Q&A]
I am running ord locally with JSON API enabled. When I make a call to a specific parent inscription, I only get back the first 4 children in the array of the response, yet there should be 1000 children for the specific parent inscription. How can I get a response for ALL children inscription id's?

**Comment by raphjaph** [comment]
`/r/children/<PARENT_ID>`

https://docs.ordinals.com/inscriptions/recursion.html
